src/lib/storage/BaseRecord.ts

```diff
@@ -4,6 +4,7 @@ export enum RecordType {
   BasicMessageRecord = 'BasicMessageRecord',
   ProvisioningRecord = 'ProvisioningRecord',
   CredentialRecord = 'CredentialRecord',
+  ProofRecord = 'ProofRecord',
```

Comment:
Maybe use `PresentationRecord` instead?

@jakubkoci What do you think?

Reply:
changed to 'PresentationRecord'

Reply:
To be honest, I'm a little bit confused.

If we look at https://github.com/hyperledger/aries-rfcs/blob/master/features/0036-issue-credential/README.md, the `credential` is subject and everything is named accordingly. I would expect a similar naming in https://github.com/hyperledger/aries-rfcs/blob/master/features/0037-present-proof/README.md, where the proof is subject. But message type names using the word `presentation` instead of `proof`. 

.NET project is using the word `presentation` in message types, but the word `proof` for the name of record, service, and so on. Although, I don't have a strong opinion I like that approach.

So I would suggest using `ProofsModule`, `ProofService`, `ProofRepository`, `ProofRecord`.

Reply:
`ProofsModule` is with `s` to be consistent with other module names (connections, credentials, ...).

Reply:
I agree with your proposal @jakubkoci

Reply:
Done

Reply:
Done